src/qml/QFieldCloudScreen.qml

```diff
@@ -1117,4 +912,8 @@ Page {
       header.onFinished();
     }
   }
+
+  function pushProjectChanges(projectId, shouldDownloadUpdates) {
```

Comment:
Do we realy need a function one(a, b) { b(a, b); } here? :) Let's just call b() directly instead where it's needed.